tests/test_examples.py

```diff
@@ -1630,3 +1632,153 @@ def circle_constraint(params):
         abs(surf_opt.Z_lmn[surf_opt.Z_basis.get_idx(M=-1, N=0)]) + offset,
         rtol=2e-2,
     )
+
+
+@pytest.mark.unit
+def test_ballooning_stability_opt():
+    """Perform ballooning stability optimization with DESC."""
+    try:
+        eq = get("HELIOTRON")[-1]
+    except TypeError:
+        eq = get("HELIOTRON")
```

Comment:
this should always be the correct one, it doesn't return a family unless you ask for it